build.gradle

```diff
@@ -18,4 +18,8 @@ allprojects {
     repositories {
         jcenter()
     }
-}
\ No newline at end of file
+}
+
+wrapper {
```

Comment:
I've never needed this before. Can you safely remove it? How have you updated the gradle version? I normally run `./gradlew wrapper  --gradle-version 4.10.2 --distribution-type all`